object-construction-checker/tests/mustcall/CommonModuleCrash.java

```diff
@@ -1,11 +1,9 @@
 import java.net.*;
 
 class CommonModuleCrash {
-    // :: error: required.method.not.called
     Socket bar = new Socket();
     static void baz(Socket s) {  }
     static {
-        // :: error: required.method.not.called
```

Comment:
why this change?

Reply:
These were false positives - these sockets are unconnected.

Reply:
same reason